### DISCUSSION TITLE
Let integrate SubWallet and SubConnect to help user easily select all Wallet in Dotsama Ecosystem

### DISCUSSION THREAD

**Discussion by hieudd** [darwinia-network/darwinia#852] [Ideas]
The wallet is the gateway to the Web3 world, I suggest integrating more user-friendly wallets like SubWallet. It is the first extension wallet that supports EVM Assets management in the Polkadot ecosystem. Polkadot{.js} extension is very unfriendly and it doesn't support EVM Assets.

- Website: https://subwallet.app/
- Twitter: https://twitter.com/subwalletapp
- Chrome, Brave, Edge: https://bit.ly/3BGqFt1
- Firefox: https://mzl.la/3rQ0awW

I refer to the SubConnect module, we built a Mini app to connect all wallets in the Polkadot ecosystem which includes Polkadot{.js}, SubWallet, and Talisman.

- Demo link: https://connect.subwallet.app/
- Source Code: https://github.com/Koniverse/SubConnect
- Video Demo how it works: 

https://www.loom.com/share/3248348dafef4b28aee365b90e1e0b16

Currently, I have not seen a library that fully supports wallets in the Dotsama ecosystem like SubConnect. Projects seem a bit confused to integrate multiple wallets, that's why we built it

![Screen Shot 2022-02-25 at 1 07 12 AM](https://user-images.githubusercontent.com/2582652/155582231-053f003e-2932-47da-ad49-c3a6e4f8e0cf.png)

**Comment by aurexav** [comment]
cc @1022ranji @JayJay1024

**Comment by hackfisher** [comment]
I think this specific integration good for some applications that need to be access from users in Polkadot ecosystem, so I moved to the issues list for para.link app.

https://github.com/ringecosystem/paralink-ui/issues/55
